packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Shared/ModelSerializationHelper.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Generator.CSharp
+{
+    internal static class ModelSerializationHelper
```

Comment:
Could we also add the AdditionalProperties const and scope this class to AdditionalProperties?